.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v*.*.*'
+      # to be used by fork patch-releases ^^
+      - 'v*.*.*-*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@master
+        with:
+          go-version: 1.16.x
```

Comment:
I have one suggestion here, since I see this popping up:
Why don't we bump the version of golang within the `go.mod`, since it's set to `1.14`, but on our docs it's noted that a version `>=1.16` is required?